appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.11.0",
+    "version": "0.11.1",
```

Comment:
No need to bump the version. Its already bumped on master and we'll include this change before publishing